service.js

```diff
@@ -1,8 +1,7 @@
 process.setMaxListeners(0);
 require('events').EventEmitter.prototype._maxListeners = 100;
-
-if (process.env.NEW_RELIC_ENABLED === 'true') require('newrelic');
-
+let newrelic;
+if (process.env.NEW_RELIC_ENABLED === 'true') newrelic = require('newrelic');
```

Comment:
you could just do `const newrelic = require('newrelic);` as `process.env.NEW_RELIC_ENABLED` is checked in the newrelic library itself